library/src/main/java/no/finntech/capturandro/Capturandro.java

```diff
@@ -89,7 +88,6 @@ public static String getFileProviderAuthority() {
      * When executing subscribe on the Obserable<Uri> image processing may take place (on a seperate thread).
      * Image processing is done on a single background thread to prevent oom
      */
-    @RequiresPermission(allOf = { Manifest.permission.WRITE_EXTERNAL_STORAGE, Manifest.permission.CAMERA })
```

Comment:
Maybe we should have kept the permission annotation?
`@RequiresPermission(Manifest.permission.CAMERA)`

(In that case, same with the function that calls this one)